### PR TITLE
Update Phase endpoint

### PR DESCRIPTION
**Pull request by harry-kim**
Added is_custom
Removed some remaining persist_open_tasks
Removed active from param list as it is currently read-only

Refs: #SDE-3641

TODO:
- [x] Set milestone label in pull request (SDE version)
- [x] Updated changelog.md
- [x] Completed documentation for the new endpoint.
- [x] Feature has been merged into SDE develop.

### CODE REVIEW

File: source/includes/phases.md [hunk 1]
```diff
@@ -180,8 +180,8 @@ name               | Yes      | The name of the phase.
 description        | Yes      | The description of the phase.
 tip                | Yes      | The tooltip for the phase.
 ordinal            | Yes      | The order of the phase.
-active             | No      | Whether or not the phase is active.
-retain             | Yes      | Determines whether this phase carries over statuses and notes by default.
+active             | No       | Whether or not the phase is active.
+retain             | No       | Determines whether this phase carries over statuses and notes by default.
```
Comment (rashidg) [inline source/includes/phases.md]: Shouldn't PUT require everything?

Comment (harry-kim) [inline source/includes/phases.md]: no because they have defaults. should be same as POST which was wrong but fixed now

**Comment by aeldamaty** [comment]
+1

**Comment by RouganStriker** [comment]
+1

**Pull request merged**